model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -247,16 +248,52 @@ def _create_concurrency_list(self) -> List[int]:
                 self._cli_config.run_config_search_max_concurrency,
             )
 
-    def _create_periodic_concurrency_list(self) -> List[int]:
+    def _create_periodic_concurrency_list(self) -> List[str]:
         if self._model_parameters["periodic_concurrency"]:
             return sorted(self._model_parameters["periodic_concurrency"])
```

Comment:
Does this also return `List[str]`?

Reply:
Yes `self._model_parameters["periodic_concurrency"]` returns a List[str]